.jshintrc

```diff
@@ -5,9 +5,9 @@
 "eqnull" : true,
 "indent": 2,
 "undef": true,
-"unused": true,
-"quotmark": "single",
-"maxlen": 90,
+"unused": false,
+"quotmark": "true",
+"maxlen": 110,
```

Comment:
In my experience, long lines are difficult to review on GitHub, because they don't fit into the code viewport and one has to scroll horizontally to see the whole line. Having said that, I don't remember what's the exact size of the viewport. If 110 characters are displayed without scrolling, then this is fine.